### PR TITLE
Add script to derive version tag for artifacts

### PR DESCRIPTION
**Pull request by odyssey4me**

### CODE REVIEW

File: derive-artifact-version.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+CMD_GET_BRANCH =(
+    "git branch --contains $(git rev-parse HEAD) | grep ^\* | sed 's/^\* //'"
+)
+
+def main():
+    """Run the main application."""
+
+    current_tag = subprocess.check_output(
+                      CMD_GET_TAG, cwd=GIT_FOLDER, shell=True
+                  ).strip(' \n\t')
```
Comment (nrb) [inline derive-artifact-version.py]: If you want all whitespace, just 'strip()' should be sufficient.

Comment (odyssey4me) [inline derive-artifact-version.py]: Does 'whitespace' include line breaks and carriage returns? I specifically want to remove all of those things.

**Comment by odyssey4me** [comment]
@nrb The plan is to use this in order to produce the artifacts for whatever is the next release. Once we have them publishing we'll be starting work in rpc-openstack to make use of the artifacts.

File: derive-artifact-version.py (hunk 1 above)
Comment (nrb) [inline derive-artifact-version.py]: Yep

Comment (BjoernT) [inline derive-artifact-version.py]: This would be a fine ansible module ;-)

**Comment by odyssey4me** [comment]
@nrb I confirmed that strip() does the required thing, so the script's been adjusted.

**Pull request merged**